src/authenticators/mod.rs

```diff
@@ -24,7 +24,10 @@ use zeroize::Zeroize;
 
 /// String wrapper for app names
 #[derive(Debug, Clone, Eq, PartialEq, Hash)]
-pub struct ApplicationName(String);
+pub struct ApplicationName {
```

Comment:
Wasn't sure if I should just change this to `Application`

Reply:
I think for this PR this is fine keeping the name to not trigger too many changes